src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,67 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        int[] output = readFile(fromFileName);
+        String result = getResult(output);
+        writeFile(result, toFileName);
+    }
+
+    public void writeFile(String result, String toFileName) {
+        BufferedWriter writer = null;
+            try {
+                writer = new BufferedWriter(new FileWriter(toFileName, true));
+                writer.write(result);
+            } catch (IOException e) {
+                throw new RuntimeException("Can't write data to file", e);
+            } finally {
+                try {
+                    writer.close();
+                } catch (IOException e) {
+                    throw new RuntimeException(e);
+                }
+            }
```

Comment:
Use try-with-resources for the BufferedWriter to ensure that resources are closed properly even if an exception occurs.